src/newton1d.jl

```diff
@@ -0,0 +1,99 @@
+doc"""`newton1d` performs the interval Newton method on the given function `f`
```

Comment:
I believe that `doc` is not necessary here. 
It is/was necessary when using `$` for LaTeX strings, but this can be avoided using double ``